common/test/acceptance/tests/studio/test_studio_asset.py

```diff
@@ -1,19 +1,25 @@
 """
 Acceptance tests for Studio related to the asset index page.
 """
+import os
+from path import Path
 
 from common.test.acceptance.fixtures.base import StudioApiLoginError
-from common.test.acceptance.pages.studio.asset_index import AssetIndexPage
+from common.test.acceptance.fixtures.config import ConfigModelFixture
+from common.test.acceptance.pages.studio.asset_index import AssetIndexPage, AssetIndexPageStudioFrontend
 from common.test.acceptance.tests.helpers import skip_if_browser
 from common.test.acceptance.tests.studio.base_studio_test import StudioCourseTest
 
+UPLOAD_SUFFIX = '/data/uploads/studio-uploads/'
+UPLOAD_FILE_DIR = Path(__file__).abspath().dirname().dirname().dirname().dirname() + UPLOAD_SUFFIX  # pylint: disable=no-value-for-parameter
```

Comment:
Can `UPLOAD_FILE_DIR` be imported from `common.test.acceptance.pages.studio.asset_index` instead?

Reply:
👍